Source/automap.cpp

```diff
@@ -516,15 +516,15 @@ void InitAutomap()
 	for (unsigned i = 1; i <= dwTiles; i++) {
 		uint8_t b1 = *pTmp++;
 		uint8_t b2 = *pTmp++;
-		automaptype[i] = b1 + (b2 << 8);
+		AutomapTypes[i] = b1 + (b2 << 8);
 	}
 
 	mem_free_dbg(pAFile);
 	memset(automapview, 0, sizeof(automapview));
 
 	for (int y = 0; y < MAXDUNY; y++) {
-		for (auto &dFlag : dFlags)
-			dFlag[y] &= ~BFLAG_EXPLORED;
+		for (int x = 0; x < MAXDUNX; x++)

```

Comment:
@glebm do you know how we could avoid it formatting 2D arrays into partially using ranges?

Reply:
Can be disabled with a magic comment:
a) At the end of the line `// NOLINT(modernize-loop-convert)`
b) On the line before`// NOLINTNEXTLINE(modernize-loop-convert)`